Package.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version: 5.8
+
+import PackageDescription
+
+private let packageName = "SimpleNetworkService"
+private let networkService = "NetworkService"
```

Comment:
nit:

```suggestion
private let networkServiceTargetName = "NetworkService"
```

or 

```suggestion
private let mainTargetName = "NetworkService"
```

?

Reply:
Please review the rest of code.

Reply:
I will choose the first option, it seems to me that with the addition of new targets it will look better.